src/vs/workbench/services/suggest/browser/simpleSuggestWidget.ts

```diff
@@ -141,6 +143,9 @@ export class SimpleSuggestWidget extends Disposable {
 			state = undefined;
 		}));
 
+		const applyIconStyle = () => this.element.domNode.classList.toggle('no-icons', !configurationService.getValue('editor.suggest.showIcons'));
```

Comment:
Should we have our own version of this setting that falls back like fontFamily does?

Reply:
trying to decrease the count of issues 😢 so maybe we can wait and see if ppl request a separate setting?